posthog/hogql/functions/mapping.py

```diff
@@ -568,6 +568,10 @@ class HogQLFunctionMeta:
     "nth_value": HogQLFunctionMeta("nth_value", 2, 2),
     "lagInFrame": HogQLFunctionMeta("lagInFrame", 1, 1),
     "leadInFrame": HogQLFunctionMeta("leadInFrame", 1, 1),
+    # dict
+    "dictGet": HogQLFunctionMeta("dictGet", 3, 3),
+    "dictGetOrNull": HogQLFunctionMeta("dictGetOrNull", 3, 3),
+    "dictGetOrDefault": HogQLFunctionMeta("dictGetOrDefault", 4, 4),
```

Comment:
Are there any `dict`-s that we don't want the user accessing? I think these functions should get an extra sanity check inside them (in the printer?).

Reply:
That could work, but then we're exposing another internal method externally, which also isn't that ideal. E.g. what if ClickHouse one day also implements a method with the same name. It'll be [JavaScript ProtoType hell](http://prototypejs.org/learn/extensions) all over again 😅

Reply:
These are gone, I now have custom hogql functions with a `hogql_` prefix